viewing_party/party.py

```diff
@@ -1,23 +1,471 @@
+# #   - the value of `user_data` will be a dictionary with a key 
+# # `"watched"`, and a value which is a list of dictionaries representing 
+# # the movies the user has watched
+#     user_data["watched"] = movie
+#     if user_data:
+#         user_data["watched"].update(movie)
+#     else:
+#         return None
+
+# ========================================= wave 01- # 2. add to watchlist========== RC
+
+# 3. Create a function named `add_to_watchlist`. This function should...take two parameters: `user_data`, `movie`
+# def add_to_watchlist(user_data, movie):
+# #   - the value of `user_data` will be a dictionary with a key `"watchlist"`, and a value which is a list of dictionaries representing the movies the user wants to watch
+#     user_data["watchlist"] = movie
+#     if user_data:
+#         user_data["watchlist"].update(movie)
+#         return user_data
+#     # return An empty list represents that the user has no movies in their watchlist
+#     else:
+#         return None
+
+# ========================================= wave 01- # 2. add to watched ========== SJ
+
+# take two parameters: user_data, movie 
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+
+    # print(user_data)
+    return user_data
+
+# ========================================= wave 01- # 3. add to watchlist ========== SJ
+# take two parameters: user_data, movie
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+
+# ========================================= wave 01- # 4. watch movie ========== RC
+
+# def watch_movie(user_data, title):
+#     # - the value of `user_data` will be a dictionary with a `"watchlist"` and a `"watched"`   
+#     # This represents that the user has a watchlist and a list of watched movies
+#     # The value of `title` will be a string
+#     # This represents the title of the movie the user has watched
+#     for movie in user_data["watchlist"]:
+#     # - If the title is in a movie in the user's watchlist:
+#         # if
+#         # remove that movie from the watchlist
+#         user_data["watchlist"].pop(movie)
+#         # add that movie to watched
+#         user_data["watched"].append(movie)
+#         # return the `user_data`
+#         return user_data
+#     # - If the title is not a movie in the user's watchlist:
+#         # else:
+#             # return the `user_data`    
+#             # return user_data
+            
+# ========================================= wave 01- # 4. watch movie ========== RC
+
+def watch_movie(user_data, title):
+    # if there's a movie dictionary in watchlist with the title fleabag, remove it from watchlist and add it to watched
+    for movie in user_data["watchlist"]:
+    # check the title for each element
+        if title == movie["title"]:
+            user_data["watchlist"].remove(movie)
+            user_data["watched"].append(movie)
+            print(user_data)  

```

Comment:
It’s good practice to avoid committing debugging log statements when submitting a PR. Remove your comments and print statements then resubmit 👍🏾